apps/change-log.md

```diff
@@ -10,6 +10,10 @@ In order to keep track of these changes we recommend you to follow [@epagesdevs]
 
 | Date      | Description      | Available with software release  |
 |---------------|---------------| -------|----------|
+| 2016&#8209;02&#8209;08 | Added new endpoint: | 6.17.39 |
+| |[`GET` shop information](page:apps-api-get-shops-shopid-information) | |
+| | Added new attributes `shippingData` and `paymentData` to orders. | |
+| | Fixed/added attributes `customerComment` and `internaNote` to orders. | |
```

Comment:
maybe: replaced attribute "comment" with attributes "customerComment" and "internaNote"